specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizerReceipt.json

```diff
@@ -47,37 +48,32 @@
         ],
         "responses": {
           "202": {
-            "description": "The service has accepted the request and will start processing later. It will return 'Accepted' immediately and include an 'Operation-Location' header. Client side should further query the operation status using the URL specified in this header. The 'Operation-Location' URL will expire in 48 hours.",
+            "description": "The service has accepted the request for processing.  The client can query the operation status and retrieve the analysis result using the 'Operation-Location' URL header specified in the response. Note that the 'Operation-Location' URL expires in 48 hours.",
             "headers": {
               "Operation-Location": {
-                "description": "URL to query for status of the operation. The URL will expire in 48 hours. ",
-                "type": "string"
+                "type": "string",
+                "description": "URL to query and retrieve the analysis result. This URL expires in 48 hours."
               }
             }
-          },
-          "default": {
```

Comment:
Why are you removing the schema information for default/non-success status codes?

Reply:
Previous reviewers indicated that explicitly listing individual error status codes (4xx, 5xx) may cause issues in the SDK generation.  While removing the individual status codes, I forgot to change it back to default.  Is it critical that we specify the schema for default response code?

One question: If we explicitly setting the schema for default status code, it means that we cannot have different schemas across error codes.  Though that is currently not an issue in our current implementation, it feels odd.  What is the guideline here?

Reply:
The general guideline is to (assuming that you are using the same schema for non-success status code - which the vast majority of services do), provide said schema in the "default" section.